CHANGELOG.md

```diff
@@ -4,6 +4,15 @@
 
 This project follows [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [v8.6.4] - 2024-11-25
+
+### Added
+
```

Comment:
* Fix the decoding of alarm_type_supported_report when version 1 of alarm types and events are also supported by the device ([#1024](https://github.com/smartrent/grizzly/pull/1024))
```